fastlane/Fastfile

```diff
@@ -767,11 +787,6 @@ def send_message_to_mattermost(options)
                          title: 'Build Number',
                          value: options[:build_number],
                          short: true
-                     },
-                     {
-                         title: 'Built by',
-                         value: 'Jenkins',
-                         short: true
```

Comment:
I believe Jenkins is no longer accurate. Or if there's still an option for it, we may add `CIRCLECI` env to better show where it's built.

Reply:
This is no longer Jenkins. There is `CIRCLE_BUILD_URL` https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables, if we can add easily env vars.

Reply:
Yeah, I can replace it with build URL if we want to show it in the webhook message.

Reply:
are  you going to change this now?

Reply:
Okay, I think I'll submit a separate PR for it.